Fix these warnings:

  CC      lib/radix-tree.o                                                                                                                                                     
lib/radix-tree.c:91: warning: excess elements in array initializer                                                                                                             
lib/radix-tree.c:91: warning: (near initialization for 'radix_node_key_string')                                                                                                
lib/radix-tree.c:92: warning: excess elements in array initializer                                                                                                             
lib/radix-tree.c:92: warning: (near initialization for 'radix_node_key_string')                                                                                                
lib/radix-tree.c:93: warning: excess elements in array initializer                                                                                                             
lib/radix-tree.c:93: warning: (near initialization for 'radix_node_key_string')                                                                                                
lib/radix-tree.c:94: warning: excess elements in array initializer                                                                                                             
lib/radix-tree.c:94: warning: (near initialization for 'radix_node_key_string')                                                                                                
lib/radix-tree.c:95: warning: excess elements in array initializer                                                                                                             
lib/radix-tree.c:95: warning: (near initialization for 'radix_node_key_string')                                                                                                
lib/radix-tree.c:96: warning: excess elements in array initializer                                                                                                             
lib/radix-tree.c:96: warning: (near initialization for 'radix_node_key_string')                                                                                                
lib/radix-tree.c:97: warning: excess elements in array initializer                                                                                                             
lib/radix-tree.c:97: warning: (near initialization for 'radix_node_key_string')                                                                                                
lib/radix-tree.c:98: warning: excess elements in array initializer                                                                                                             
lib/radix-tree.c:98: warning: (near initialization for 'radix_node_key_string')                                                                                                
lib/radix-tree.c:99: warning: excess elements in array initializer                                                                                                             
lib/radix-tree.c:99: warning: (near initialization for 'radix_node_key_string')                                                                                                
lib/radix-tree.c:83: warning: 'radix_node_key_string' defined but not used 

Signed-off-by: Remy Bohmer <linux@bohmer.net>
---
 lib/radix-tree.c |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

Index: linux-2.6.26/lib/radix-tree.c
===================================================================
--- linux-2.6.26.orig/lib/radix-tree.c	2009-07-26 15:18:11.000000000 +0200
+++ linux-2.6.26/lib/radix-tree.c	2009-07-26 15:20:11.000000000 +0200
@@ -78,7 +78,6 @@ static unsigned long height_to_maxindex[
 
 #ifdef CONFIG_RADIX_TREE_CONCURRENT
 static struct lock_class_key radix_node_class[RADIX_TREE_MAX_PATH + 1];
-#endif
 #ifdef CONFIG_DEBUG_LOCK_ALLOC
 static const char *radix_node_key_string[RADIX_TREE_MAX_PATH + 1] = {
 	"radix-node-00",
@@ -88,6 +87,7 @@ static const char *radix_node_key_string
 	"radix-node-04",
 	"radix-node-05",
 	"radix-node-06",
+#ifdef CONFIG_BASE_SMALL
 	"radix-node-07",
 	"radix-node-08",
 	"radix-node-09",
@@ -97,8 +97,10 @@ static const char *radix_node_key_string
 	"radix-node-13",
 	"radix-node-14",
 	"radix-node-15",
+#endif
 };
 #endif
+#endif
 
 #ifdef CONFIG_RADIX_TREE_OPTIMISTIC
 static DEFINE_PER_CPU(unsigned long[RADIX_TREE_MAX_PATH+1], optimistic_histogram);
